core/tee/tee_obj.c

```diff
@@ -70,10 +70,11 @@ void tee_obj_close(struct user_ta_ctx *utc, struct tee_obj *o)
 
 void tee_obj_close_all(struct user_ta_ctx *utc)
 {
-	struct tee_obj_head *objects = &utc->objects;
+	struct tee_obj *o;
+	struct tee_obj *next;
 
-	while (!TAILQ_EMPTY(objects))
-		tee_obj_close(utc, TAILQ_FIRST(objects));
```

Comment:
@jforissier It should be possible to submit a model to ignore the false positives. I'll take a closer look and update the Coverity project with it as soon as possible. To ignore these particular occurrences, we can just go to the defects list, select the CIDs and choose Ignore for the Action.